privacy-security-questionnaire.md

```diff
@@ -0,0 +1,76 @@
+## Self-Review Questionnaire: Security and Privacy
+
+| 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This feature is a Document-Policy header that an origin may choose to include in an HTTP resonse when it has no linked resources in the response, to indicate to the user agent that it may optimize resources required to conduct a speculative parse of the response. This will benefit the user in terms of reducing compute requirements on the client, and improve performance of the experience.
+
+| 2.2. Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
+
+Yes. The feature only exposes the minimum required information as a boolean header, to satisfy the use case.
+
+| 2.3. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
+Personal information is any data about a user (for example, their home address), or information that could be used to identify a user, such as an alias, email address, or identification number.
```

Comment:
I don't think restating the definition of PI is helpful here.

Reply:
Ouch. that was a copy paste error. Line 14 was supposed to be the answer :)